positionpaper.bib

```diff
@@ -30,4 +30,9 @@ @misc{fair_data_maturity_model_working_group_2020
   version      = {1.0},
   doi          = {10.15497/rda00050},
   url          = {https://doi.org/10.15497/rda00050}
+}
+
+@misc{manchester_mobile,
```

Comment:
did you try whether @ online makes more sense?

Reply:
I did not know about `@online`...